### DISCUSSION TITLE
How do I get the designer's code, because I want to change something

### DISCUSSION THREAD

**Discussion by landonzeng** [elsa-workflows/elsa-core#2539] [Q&A]
How do I get the designer's code, because I want to change something

**Comment by ghost** [comment]
https://github.com/elsa-workflows/elsa-dashboard-blazor 

designer source code in github, kindly elaborate the question if you are asking for something else

Best Regards

**Comment by landonzeng** [comment]
> https://github.com/elsa-workflows/elsa-dashboard-blazor
> 
> designer source code in github, kindly elaborate the question if you are asking for something else
> 
> Best Regards

I would like to modify the language on the page based on the current design, as one of my clients requires the display in Chinese

**Comment by ghost** [comment]
no worries, is this what you are looking for?
https://github.com/elsa-workflows/elsa-dashboard-blazor

wish you happy coding :)

**Comment by landonzeng** [comment]
> no worries, is this what you are looking for? https://github.com/elsa-workflows/elsa-dashboard-blazor wish you happy coding :)
> […](#)
> On Thu, Nov 18, 2021 at 9:57 AM landon ***@***.***> wrote: https://github.com/elsa-workflows/elsa-dashboard-blazor designer source code in github, kindly elaborate the question if you are asking for something else Best Regards I would like to modify the language on the page based on the current design, as one of my clients requires the display in Chinese — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#2484 (comment)](https://github.com/elsa-workflows/elsa-core/issues/2484#issuecomment-972588868)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AKECRBZULEJ236O3I4NLOTDUMSPVXANCNFSM5IDE34MQ> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

I'd like to modify the code in the Elsa.Designer.Components.Web class library referenced in elsa, but I don't know if I'm describing it correctly，According to the documentation provided by elsa, the dashboard interface is designed to have pages by referring to this library, so I want to modify the language of all pages in this dashboard

**Reply by elloku** [reply]
活动节点的中文翻译做出来了嘛，我看活动节点的名称和描述都是在代码里面固定的，好像没办法使用资源替换的形式实现中文界面

**Reply by landonzeng** [reply]
> 活动节点的中文翻译做出来了嘛，我看活动节点的名称和描述都是在代码里面固定的，好像没办法使用资源替换的形式实现中文界面

节点没法翻译，好像是后端编码写死的

**Comment by sfmskywalker** [comment]
Hi everyone,

[Elsa Studio](https://github.com/elsa-workflows/elsa-studio) now supports localization, and we're looking for contributors to help create translations. We use [Weblate](https://hosted.weblate.org/projects/elsa-studio/) to make managing translations simple and collaborative. If you're interested in contributing Chinese translations, please sign up for a Weblate account and share your username or email address with me.

We'd greatly appreciate your help!

**Reply by joskoanicic** [reply]
> Hi everyone,
> 
> [Elsa Studio](https://github.com/elsa-workflows/elsa-studio) now supports localization, and we're looking for contributors to help create translations. We use [Weblate](https://hosted.weblate.org/projects/elsa-studio/) to make managing translations simple and collaborative. If you're interested in contributing Chinese translations, please sign up for a Weblate account and share your username or email address with me.
> 
> We'd greatly appreciate your help!

Are there any “getting started” instructions available, as we are new to using Weblate? We are planning to implement a few languages in the coming months for our needs, so contributing back would be a great way to collaborate.

**Reply by sfmskywalker** [reply]
I don't have anything available, so just a quick few pointers:

1. Sign up for an account with Weblate and provide me with your username. Alternatively, give me your email address and I'll invite you from there.
2. Once signed in, find your way to creating new languages and start translating the base strings. The UI is pretty straightforward, but let me know if you can't find your way.

https://hosted.weblate.org/projects/elsa-studio/elsa-studio-translations/
